app/lib/submission_builder/ty2024/states/md/documents/md502_su.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module SubmissionBuilder
+  module Ty2024
+    module States
+      module Md
+        module Documents
+          class Md502Su < SubmissionBuilder::Document
+            include SubmissionBuilder::FormattingMethods
+
+            def document
+              build_xml_doc("Form502SU", documentId: "Form502SU") do |xml|
+                xml.Subtractions do |subtractions|
+                  subtractions.Total calculated_fields.fetch(:MD502_SU_LINE_1)
```

Comment:
[pebble] is there supposed to be an `=` here or is `Total` a method name receiving a parameter?

Reply:
this is intentional! the `build_xml_doc` method generates the xml for the state return. therefore the value for Total will be assigned from this method.